src/main/java/org/gitlab4j/api/MilestonesApi.java

```diff
@@ -21,6 +21,240 @@ public MilestonesApi(GitLabApi gitLabApi) {
+     * @param state the milestone state
+     * @return the milestones associated with the specified group and state
+     * @throws GitLabApiException if any exception occurs
+     */
+    public List<Milestone> getGroupMilestones(Object groupIdOrPath, MilestoneState state) throws GitLabApiException {
+        Form formData = new GitLabApiForm().withParam("state", state).withParam(PER_PAGE_PARAM, getDefaultPerPage());
+        Response response = get(Response.Status.OK, formData.asMap(),
+                "groups", getGroupIdOrPath(groupIdOrPath), "milestones");
+        return (response.readEntity(new GenericType<List<Milestone>>() {}));
+    }
+
+    /**
+     * Get a list of group milestones that have match the search string.
+     *
+     * @param groupIdOrPath the group in the form of an Integer(ID), String(path), or Group instance
+     * @param search the search string
+     * @return the milestones associated with the specified group
+     * @throws GitLabApiException if any exception occurs
+     */
+    public List<Milestone> getGroupMilestones(Object groupIdOrPath, String search) throws GitLabApiException {
+        Form formData = new GitLabApiForm().withParam("search", search).withParam(PER_PAGE_PARAM, getDefaultPerPage());
+        Response response = get(Response.Status.OK, formData.asMap(),
+                "groups", getGroupIdOrPath(groupIdOrPath), "milestones");
+        return (response.readEntity(new GenericType<List<Milestone>>() {}));
+    }
+
+    /**
+     * Get a list of group milestones that have the specified state and match the search string.
+     *
+     * @param groupIdOrPath the group in the form of an Integer(ID), String(path), or Group instance
+     * @param state the milestone state
+     * @param search the search string
+     * @return the milestones associated with the specified group
+     * @throws GitLabApiException if any exception occurs
+     */
+    public List<Milestone> getGroupMilestones(Object groupIdOrPath, MilestoneState state, String search) throws GitLabApiException {
+        Form formData = new GitLabApiForm()
+                .withParam("state", state)
+                .withParam("search", search)
+                .withParam(PER_PAGE_PARAM, getDefaultPerPage());
+        Response response = get(Response.Status.OK, formData.asMap(),
+                "groups", getGroupIdOrPath(groupIdOrPath), "milestones");
+        return (response.readEntity(new GenericType<List<Milestone>>() {}));
+    }
+
+    /**
+     * Get the specified group milestone.
+     *
+     * @param groupIdOrPath the group in the form of an Integer(ID), String(path), or Group instance
+     * @param milestoneId the ID of the milestone tp get
+     * @return a Milestone instance for the specified IDs
+     * @throws GitLabApiException if any exception occurs
+     */
+    public Milestone getGroupMilestone(Object groupIdOrPath, int milestoneId) throws GitLabApiException {
```

Comment:
For consistency, ```int``` should be ```Integer```

Reply:
Done.

Just for your knowledge, the exiting [getMilestone()](https://github.com/gmessner/gitlab4j-api/blob/master/src/main/java/org/gitlab4j/api/MilestonesApi.java#L131) method also has the wrong signature.
I didn't change it as I don't manage the impacts.

Reply:
@pismy 
I will change the existing ```getMilestone()``` post-merge, unless you'd like to make the change.  It should have no impact.

Reply:
just did it.

Reply:
How often do you make releases ?

Reply:
@pismy 
I typically try and release when a PR is merged, and will usually include updates that are a result of issues being fixed or implemented.  SO you can expect a release either today or tomorrow.